test/service.ts

```diff
@@ -1682,21 +1677,9 @@ describe('GrpcService', () => {
       it('should return grpcCredentials', done => {
         grpcService.getGrpcCredentials_((err, grpcCredentials) => {
           assert.ifError(err);
-
-          assert.strictEqual(grpcCredentials.name, 'combineChannelCredentials');
-
-          const createSslArg = grpcCredentials.args[0];
-          assert.strictEqual(createSslArg.name, 'createSsl');
-          assert.deepStrictEqual(createSslArg.args.length, 0);
-
-          const createFromGoogleCredentialArg = grpcCredentials.args[1];
-          assert.strictEqual(
-            createFromGoogleCredentialArg.name,
-            'createFromGoogleCredential'
-          );
           assert.strictEqual(
-            createFromGoogleCredentialArg.args[0],
-            AUTH_CLIENT
+            grpcCredentials.constructor.name,
```

Comment:
seems like this API surface here would be a breaking change? but is this breaking the law of demeter, mainly I don't think people using `nodejs-common-grpc` would have a reason to reach in here right?